src/Agent.Worker/CodeCoverage/CoberturaSummaryReader.cs

```diff
@@ -0,0 +1,96 @@
+﻿using Microsoft.TeamFoundation.TestManagement.WebApi;
```

Comment:
what is the directory structure we follow. I see interface and impl in single file